src/component/main-content/main-content.js

```diff
@@ -8,16 +9,13 @@ const {func, object, arrayOf, shape, array} = React.PropTypes;
 
 export default class MainContent extends React.Component {
 
-  constructor() {
-    super();
-  }
-
-  componentWillMount() {
-    this.setState({
+  constructor(props) {
```

Comment:
Move state setting to constructor, as laid out here: https://facebook.github.io/react/docs/reusable-components.html#es6-classes